src/Util/Crypto.php

```diff
@@ -318,13 +318,13 @@ private static function decryptAES256CTR($data, $key, $iv)
 	}
 
 	/**
-	 * 
+	 *
 	 * Ported from Hubzilla: https://framagit.org/hubzilla/core/blob/master/include/crypto.php
-	 * 
+	 *
 	 * @param string $data
 	 * @param string $pubkey The public key.
-	 * @param string $alg    The algorithm used for encryption.
```

Comment:
Those white spaces are actually wanted, so all the parameter comments are aligned.